### PR TITLE
Create bibliothecae-it.csl

### PR DESCRIPTION
**Pull request by POBrien333**
via https://forums.zotero.org/discussion/87588/style-request-bibliothecae-it#latest

### CODE REVIEW

**Comment by retorquere** [comment]
@rmzelle we should fix the problem that initiated #5438 to curb false negatives while we figure out how to address #5438.

**Comment by rmzelle** [comment]
> @rmzelle we should fix the problem that initiated #5438 to curb false negatives while we figure out how to address #5438.

Done via https://github.com/citation-style-language/styles/pull/5444. Sorry about that.

File: bibliothecae-it.csl [hunk 1]
```diff
@@ -0,0 +1,246 @@
...
+        <single>p.</single>
+        <multiple>pp.</multiple>
+      </term>
+      <term name="and others"/>
+      <term name="letter" form="short">lettera</term>
+    </terms>
+  </locale>
+  <macro name="responsibility">
+    <names variable="author">
+      <name delimiter-precedes-last="never" et-al-min="4" et-al-use-first="4"/>
```
Comment (adam3smith) [inline bibliothecae-it.csl:30]: et-al-min needs to be > et-al-use-first for logical reasons. Do you want 4/3 here or 4/1 as below?

Comment (POBrien333) [reply bibliothecae-it.csl]: thanks for spotting that error!

**Comment by rmzelle** [comment]
@POBrien333, thanks!

**Pull request merged**